triaging.rst

```diff
@@ -47,6 +47,35 @@ the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
    Existing members with the "Developer" role on the `issue tracker`_ are welcome to
    self-nominate for triage team membership if they are interested.
+   
+GitHub Labels
+''''''''''''''''
+
+An important component of triaging PRs and issues involves appropriately 
+categorizing them through the usage of labels. Here are several of the
+common ones utilized by the Python triage team:
+
++-----------------------+----------------------------------------------------------------------------+
+| Label                 | Description                                                                |
++-----------------------+----------------------------------------------------------------------------+
+| needs backport to X.Y | Used for PRs which are appropriate to backport to branches prior to        |
```

Comment:
We can add that, by adding this label, it will trigger the bot to perform auto-backport.